__tests__/cache.test.ts

```diff
@@ -112,31 +127,35 @@ describe('InMemoryCache', () => {
       scope: 'the_scope'
     };
 
-    // Test that the cache state is normal up until just before the expiry time..
-    jest.advanceTimersByTime(799);
+    // Test that the cache state is normal before we expire the data
     expect(cache.get(cacheEntry)).toStrictEqual(data);
 
     // Advance the time to just past the expiry..
-    jest.advanceTimersByTime(1);
+    const dateNowStub = jest.fn(() => now + 86500 * 1000);
```

Comment:
Is this suggestion correct?
```suggestion
    const dateNowStub = jest.fn(() => now + dayInSeconds * 1000);
```

Reply:
It would have to be `dayInSeconds + 100` to match. The idea was that the stub would report a time between the expiry time and the `exp` value from the token, set at `dayInSeconds` and `dayInSeconds * 2`.

Reply:
Then why not use `dayInSeconds + 100` and add a comment as you did on other tests on this same diff explaining that `// take off a small leeway`. (in this case, would be 'add'). Or if you're confident it should be between the value and its double, you could also multiply by `1,5`? Anyway. I'm suggesting this because on the rest of the code you're using that _constant_ and suddenly coming across a magic number makes a bit of noise.

Reply:
Yes I'm agreeing with you, that would be a good change. Just wasn't sure if you wanted me to explain it so you can update the suggestion :)

Reply:
Hehe. Yes! Sorry for the messaging. Thanks for agreeing to change that!

Reply:
@lbalmaceda This and the other instance have been updated with this change.